Example/ViewController.swift

```diff
@@ -81,6 +81,18 @@ final class ViewController: UIViewController {
     @IBAction func stop(_ sender: UIButton) {
         player.stop()
     }
+    
+    @IBAction func metadata(_ sender: Any) {
+        let formatter = DateFormatter()
+        formatter.dateFormat = "HH:mm:ss"
+        let timeStamp = formatter.string(from: Date())
+        let newMetadata = ModernAVPlayerMediaMetadata(title: "Updated title (\(timeStamp))",
```

Comment:
hard to easily read all title string in command center (scrolling thing...)
maybe just set timestamp in the title parameter

Reply:
done